### DISCUSSION TITLE
Set language for speechRecognition

### DISCUSSION THREAD

**Discussion by h4de5** [home-assistant/frontend#7860] [Localization]
Hello,

it seems the conversions dialog only understands english - it should be possible to set the current frontend or browser language like this:

https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition/lang

when opening the dialog here:

https://github.com/home-assistant/frontend/blob/0e9984413cda2a214a12636265b7d52a008cd565/src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts#L242

https://github.com/home-assistant/home-assistant-js/blob/465f1450560cbf0aa4bd8b0cff502b95d11fa7f3/src/modules/voice/actions.js#L44

the feature has been asked for multible times:

https://community.home-assistant.io/t/conversation-language-setup/195301
https://community.home-assistant.io/t/is-conversation-integration-multi-language/150580
https://community.home-assistant.io/t/conversation-multi-language/7927
https://community.home-assistant.io/t/conversation-in-french/1985
https://community.home-assistant.io/t/conversation-non-english-languages/198660

**Comment by slydiman** [comment]
Maybe duplicate this request in [Lovelace](https://github.com/home-assistant/frontend/discussions/categories/lovelace) and/or [Configuration](https://github.com/home-assistant/frontend/discussions/categories/configuration) category?

**Reply by spacegaier** [reply]
Please don't. That does not help.

**Comment by h4de5** [comment]
Get more people to upvote it.
